com.brave.Brave.metainfo.xml

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright 2017 the Chromium authors, with modifications -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.brave.Brave</id>
+  <launchable type="desktop-id">com.brave.Brave.desktop</launchable>
+  <name>Brave Browser</name>
+  <developer_name>Brave Software</developer_name>
+  <summary>Community-maintained unofficial Flatpak of Brave Browser</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
+  <url type="homepage">https://brave.com/</url>
+  <description>
+   <p>
+      Browse faster by blocking ads and trackers that violate your privacy and cost
+      you time and money.
+   </p>
+   <p>
```

Comment:
This needs explicit disclaimer like https://flathub.org/apps/details/com.jetbrains.PyCharm-Professional has.

Reply:
Should I resolve it now?